libraries/AP_BoardConfig/AP_BoardConfig.h

```diff
@@ -205,6 +205,7 @@ class AP_BoardConfig {
     bool spi_check_register(const char *devname, uint8_t regnum, uint8_t value, uint8_t read_flag = 0x80);
     void validate_board_type(void);
     void board_autodetect(void);
+    bool check_ms5611(const char* devname);
```

Comment:
This is dangling in the case that `HAL_VALIDATE_BOARD` isn't set, right?

Should we pragma this out in here in the case that HAL_VALIDATE_BOARD isn't set?

Reply:
what do you mean ? it is only used on chibios build and all are defining HAL_VALIDATE_BOARD. If check_ms5611 isn't use it will be removed